dm/templates/instance/examples/instance.yaml

```diff
@@ -14,7 +14,8 @@ resources:
       diskImage: projects/ubuntu-os-cloud/global/images/family/ubuntu-1804-lts
       machineType: f1-micro
       diskType: pd-ssd
-      network: default
```

Comment:
Please update your implementation in a way that both 'network' (as the current implementation) as 'networks' (the new implementation) is supported to assure backwards compatibility.